thirdparty_builtin/googletest-release-1.8.0/CMakeLists.txt

```diff
@@ -16,13 +16,21 @@ project( googletest-distribution )
 
 enable_testing()
 
-option(BUILD_GTEST "Builds the googletest subproject" OFF)
+
+##########################################################
```

Comment:
I'll look into it -- Note: We already have a BLT specific fix in this file on master, we should try to move that as well.